portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,48 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Victor Alvarez - Home</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body>
+    <!-- Navigation bar from W3schools.com -->
+     <ul>
+        <li class="active"><a href="index.html">About Me</a></li>
+        <li><a href="resume.html">Resume</a></li>
+    </ul>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Victor Alvarez's Portfolio</h1>
+
+      <img src="/images/promPhoto.jpeg" alt="image of myself" style="display: block; margin-left: auto; margin-right: auto; height: 50%; width: 50%; border-radius: 50%;"/>
```

Comment:
For readability, we have this idea that your line lengths shouldn't be greater than 80 characters. I don't think GitHub shows the characters, but I have to scroll to see the entire line (unlike on likes 22-24, where you did a good job splitting things up there), so I would split up line 22.

For HTML, what you can do is put each attribute on it's own line. So it would look something like:

<img src="/images/promPhoto.jpeg"
    alt="image of myself"
    style="display: block; margin-left: auto; margin-right: auto; height: 50%; width: 50%; border-radius: 50%;"/>

That being said, I have another comment on this line about css classes, and I think after you resolve that, you shouldn't need to split this line up.

Reply:
Oops, looks like it interpreted the img tag and messed things up, let me try that again

Reply:
img src="/images/promPhoto.jpeg"
----alt="image of myself"
----style="display: block; margin-left: auto; margin-right: auto; height: 50%; width: 50%; border-radius: 50%;

The "-' above represent spaces, looks like comment formatting is weird and won't keep them